datafusion/optimizer/src/analyzer/mod.rs

```diff
@@ -72,7 +74,7 @@ pub trait AnalyzerRule {
 ///
 /// An `Analyzer` transforms a `LogicalPlan`
 /// prior to the rest of the DataFusion optimization process.
-#[derive(Clone)]
+#[derive(Clone, Debug)]
```

Comment:
It will be nice to be able to see what rules are registered with the analyzer/optimizer